internal/venice-common/src/main/java/com/linkedin/venice/controllerapi/BlobDiscoveryResponse.java

```diff
@@ -0,0 +1,17 @@
+package com.linkedin.venice.controllerapi;
+
+import java.util.List;
+
+
+public class BlobDiscoveryResponse extends ControllerResponse {
+  private boolean isError;
```

Comment:
We need getter/setter for `isError` too.

Reply:
[see comment ](https://github.com/linkedin/venice/pull/980/files#r1600163060)